src/main/java/com/gsheet/student/dto/StudentDto.java

```diff
@@ -0,0 +1,13 @@
+package com.gsheet.student.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class StudentDto {
+    private int id;
```

Comment:
Always use Object on models, as primitives don't contain `null` value, but it is a required state when we are working with DB